pkg/capr/planner/planner.go

```diff
@@ -409,10 +410,31 @@ func (p *Planner) fullReconcile(cp *rkev1.RKEControlPlane, status rkev1.RKEContr
 		return status, errWaiting("marking control plane as initialized and ready")
 	}
 
-	// Process all nodes that are ONLY worker nodes.
-	err = p.reconcile(cp, clusterSecretTokens, plan, false, workerTier, isOnlyWorker, isInitNodeOrDeleting,
-		workerConcurrency, "",
-		workerDrainOptions)
+	// Process all nodes that are ONLY linux worker nodes.
+	err = p.reconcile(cp, clusterSecretTokens, plan, false, workerTier, isOnlyLinuxWorker, isInitNodeOrDeleting,
+		workerConcurrency, "", workerDrainOptions, -1, 1,
+		false)
+	firstIgnoreError, err = ignoreErrors(firstIgnoreError, err)
+	if err != nil {
+		return status, err
+	}
+
+	// Process all nodes that are ONLY windows worker nodes.
```

Comment:
Can you expand this comment to denote why we need a higher failure threshold for Windows workers.